src/NServiceBus.ContainerTests/When_registering_components.cs

```diff
@@ -131,42 +97,6 @@ public void Setter_dependencies_should_be_supported_when_resolving_interfaces()
             }
         }
 
-
-        [Test]
-        public void Setter_dependencies_should_be_supported()
```

Comment:
>I fully agree that we no longer need to test for it. I brought it up to have a history of that discussion here.

After thinking this through I'd say we should add those tests specifically to the downstream containers since they still support property injection.

So: Copy paste the removed tests to each `UnitTest` project in the downstreams